subsys/bluetooth/audio/Kconfig.mediacontrol

```diff
@@ -0,0 +1,50 @@
+# Bluetooth Audio - Media control configuration options
+
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+config BT_MEDIACONTROL
+       bool "Support for media player and media player control"
+       help
+         Enables support for media player and for control of local and remote
+	 media players
+
+if BT_MEDIACONTROL
+
+config BT_MEDIACONTROL_LOCAL_PLAYER
+       bool "Support for local media player"
+       #TODO: Move MPL out of BT susbystem, change "select" to "depends"?
+       select BT_MPL
+       help
+         This option enables support for a local media player
+
+config BT_MEDIACONTROL_LOCAL_PLAYER_LOCAL_CONTROL
+       bool "Support for local control of local media player"
+       depends on BT_MEDIACONTROL_LOCAL_PLAYER
+       help
+         This option enables support for local application control of local
+	 media players
+
+config BT_MEDIACONTROL_LOCAL_PLAYER_REMOTE_CONTROL
+       bool "Support for remote control of local media player"
+       depends on BT_MEDIACONTROL_LOCAL_PLAYER
+       select BT_MCS
+       help
+         This option enables support for remote control of local media
+	 players, using GATT and the Media Control Service
```

Comment:
Suggest to enable this by default

Reply:
Yes :D Sorry, I had marked the wrong one. I did mean the remote controller :)

Reply:
Changed